lib/phoenix/html/tag.ex

```diff
@@ -0,0 +1,85 @@
+defmodule Phoenix.HTML.Tag do
+  @moduledoc ~S"""
+  Helpers related to producing HTML tags within templates.
+  """
+
+  alias Phoenix.HTML.Safe
+
+  @data_attrs [:method, :remote, :confirm]
+
+  @doc ~S"""
+  Creates an HTML tag with the given name and options.
+
+  ## Examples
+      iex> Phoenix.HTML.Tag.tag(:br)
+      "<br>"
+      iex> Phoenix.HTML.Tag.tag(:input, type: "text", name: "user_id")
+      "<input name="user_id" type="text">"
+  """
+  def tag(name),                              do: tag(name, [], true)
+  def tag(name, attrs) when is_list(attrs),   do: tag(name, attrs, true)
+  def tag(name, open)  when is_boolean(open), do: tag(name, [], open)
```

Comment:
Should we care about open? I think the HTML5 spec does not care about this at all anymore.